resolwe_bio/processes/workflows/bbduk_salmon_qc.py

```diff
@@ -0,0 +1,396 @@
+                required=False,
+            )
+            custom_adapter_sequences = ListField(
+                inner=StringField(),
+                label="Custom adapter sequences",
+                required=False,
+                default=[],
+                description="Custom adapter sequences can be specified by inputting them "
+                "one by one and pressing Enter after each sequence.",
+            )
+            kmer_length = IntegerField(
+                label="K-mer length",
+                default=23,
+                description="K-mer length must be smaller or equal to the length of adapters.",
+            )
+            min_k = IntegerField(
+                label="Minimum k-mer length at right end of reads used for trimming",
+                default=11,
+                disabled="preprocessing.adapters.length === 0 && preprocessing.custom_adapter_sequences.length === 0",
+            )
+            hamming_distance = IntegerField(
+                label="Maximum Hamming distance for k-mers",
+                default=1,
+            )
+            maxns = IntegerField(
+                label="Max Ns after trimming",
+                default=-1,
+                description="If non-negative, reads with more Ns than this (after trimming) will be discarded.",
+            )
+            trim_quality = IntegerField(
+                label="Quality below which to trim reads from the right end",
+                default=10,
+                description="Phred algorithm is used, which is more accurate than naive trimming.",
+            )
+            min_length = IntegerField(
+                label="Minimum read length",
+                default=20,
+                description="Reads shorter than minimum read length after trimming are discarded.",
+            )
+            quality_encoding_offset = StringField(
+                label="Quality encoding offset",
+                choices=[
+                    ("33", "Sanger / Illumina 1.8+"),
+                    ("64", "Illumina up to 1.3+, 1.5+"),
+                    ("auto", "Auto"),
+                ],
+                default="auto",
+                description="Quality encoding offset for input FASTQ files.",
+            )
+            ignore_bad_quality = BooleanField(
+                label="Ignore bad quality",
+                default=False,
+                description="Don't crash if quality values appear to be incorrect.",
+            )
+
+        class Quantification:
+            """Quantification (Salmon)."""
+
+            seq_bias = BooleanField(
+                label="Perform sequence-specific bias correction",
+                default=True,
+                description="Perform sequence-specific bias correction.",
+            )
+            gc_bias = BooleanField(
+                label="Perform fragment GC bias correction",
+                default=False,
+                description="Perform fragment GC bias correction.",
```

Comment:
This input is hardcoded as `True` for the paired-end inputs. We need to expand the description to explain the behaviour of this parameter for different input types. Since one can control this flag (disable the gc bias correction) only in the case of single-end inputs, maybe a better approach is not to set the default value to begin with. @romunov any preference?